docs/tutorials/dvm-integration.md

```diff
@@ -33,13 +33,13 @@ Details on these two fees are available [here](oracle/econ-architecture.md#step-
 2. Migrate the contracts by running the following command:
 
 ```bash
-$(npm bin)/truffle migrate --reset --network test
+yarn truffle migrate --reset --network test
 ```
 
-3. To deploy the `DepositBox` contract and go through a simple user flow, run the following script from the `/core` directory:
+3. To deploy the `DepositBox` contract and go through a simple user flow, run the following script from the `packages/core` directory:
```

Comment:
nit: I think these commands can all be run from root.

Reply:
bump

Reply:
done.